javatests/arcs/core/host/BUILD

```diff
@@ -0,0 +1,35 @@
+load(
+    "//third_party/java/arcs/build_defs:build_defs.bzl",
+    "arcs_kt_jvm_library",
+    "arcs_kt_jvm_test_suite",
+)
+
+licenses(["notice"])
+
+package(default_visibility = ["//visibility:public"])
+
+arcs_kt_jvm_test_suite(
+    name = "host",
+    srcs = ["ParticleRegistrationTest.kt"],
```

Comment:
Nit: usually I'd prefer to switch the logic around: include all `*Test.kt` sources by default, and exclude util classes listed explicitly. Or alternatively, include all `*Test.kt` and exclude all other files.

This way, if you add a new test file, you can't accidentally forget to include it in the BUILD rule.

(This probably means you have to rename the `RunInTest.kt` file.`RunInTestHost.kt` would be fine)